apps/laboratory/src/utils/ConstantsUtil.ts

```diff
@@ -23,7 +23,7 @@ if (!projectId) {
   throw new Error('NEXT_PUBLIC_PROJECT_ID is not set')
 }
 export const WALLET_URL = process.env['WALLET_URL'] || 'https://react-wallet.walletconnect.com/'
-export const USEROP_BUILDER_SERVICE_BASE_URL = 'https://rpc.walletconnect.com/v1/wallet'
+export const USEROP_BUILDER_SERVICE_BASE_URL = 'https://rpc.walletconnect.org/v1/wallet'
```

Comment:
Do we need .org ?

Reply:
@magiziz `org` is including the countries which have blocked `com` domains like China